05a-run_ica.py

```diff
@@ -127,5 +131,6 @@ def run_ica(subject, tsss=config.mf_st_duration):
             report.save(report_fname, overwrite=True, open_browser=False)
 
 
-parallel, run_func, _ = parallel_func(run_ica, n_jobs=config.N_JOBS)
-parallel(run_func(subject) for subject in config.subjects_list)
+if config.use_ica:
```

Comment:
This should issue a message if the script is run with config.use_ica = False

Reply:
I disagree here as it makes testing harder. Whatever you set in the config file then
make should just run.

Reply:
I meant that the run_ica script could inform the user that it ran but did not do any ica because the config told it not to. Still disagree?

Reply:
thx